host_v2/src/components/QuestionCard.tsx

```diff
@@ -8,36 +8,50 @@ import BodyCardStyled from '../lib/styledcomponents/BodyCardStyled';
 interface QuestionCardProps {
   questionText: string;
   imageUrl: string | undefined;
+  index: number;
 }
-
+const TitleTypography = styled(Typography)({
+  width: '100%', 
+  textAlign: 'left', 
+  fontFamily: 'Poppins',
+  fontWeight: '700', 
+  fontSize: '24px',
+  lineHeight: '36px',
+})
+const BodyTypography = styled(Typography)({
+  fontFamily: 'Rubik',
+  fontWeight: '400',
+  fontSize: '16px',
+  lineHeight: '19px',
+  color: '#384466',
+})
 export default function QuestionCard({
   questionText,
   imageUrl,
+  index,
 }: QuestionCardProps) {
   const theme = useTheme(); // eslint-disable-line
   const { t } = useTranslation();
   return (
     <BodyCardStyled elevation={10}>
       <BodyCardContainerStyled>
-        <Typography
-          variant="subtitle1"
-          sx={{ width: '100%', textAlign: 'left' }}
-        >
-          {t('gamesession.questionCard.title')}
-        </Typography>
+        <TitleTypography>
+          {t('Question ')} {index + 1}         
```

Comment:
the `t()` here is a hook used to reference our dictionary. So we'll need to add 'Question ' to our translation dictionary and then reference it (similar to line 26 that is deleted above) instead of just dropping the string into the hook here. Let me know if this doesn't make sense though

Reply:
@drewjhart

Reply:
So here's a primer on our translation set up. We are using `react-i18next` to provide support for multiple languages. How this works is that we maintain a set of dictionaries of all strings in our code in each of our supported languages. For now, we just have english and a very dumb Google translate spanish version. Then, we make use of a the `useTranslation` custom hook from the `react-18next` package to distribute the correct string based on the language settings that it determines at application runtime. 

So in this, you can see on ln 39 we  deploy the hook with: `const { t } = useTranslation();` and then we make use of it here (this has been since removed): `  {t('gamesession.questionCard.title')}`

Our dictionaries are stored in `public/locales` (one folder above `src`) and with a folder for each language (`en` for english and `es` for spanish). So, whenever you create a new string, you need to go into the dictionaries and update them with that string. Then you reference it using the hook. 

So for this:
1. Go into the english dictionary and add any new text (or make any edits if the old text just needs an update) under the correct component. In this case, it's `questionCard`. 
2. Translate the change (via either your sick spanish skills or google translate, if you're like me) and replicate the change with the correct translation in the spanish dictionary.
2. Then reference the correct path to the string in the data object (similar to how it's done on ln 26). 

This ensures that if a user hits our app with a different language (that we support), all strings will be adjusted so that it's accessible to them.